RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Onboarding changes

## Proposed changes

- Change the first screen of the app
- Minor changes on NewServerView and make it the first screen of the app
   -- This requires a few changes on the stack as well
- Add "Create workspace" to ServerDropdown
- Update E2E tests

## Issue(s)	


## How to test or reproduce

- Open the app without an account
- Enter the workspace
- Do the login flow
- When logged in:
  -  1 -> If you logout, you should be redirected to the first page
  - 2 -> If `Add a server`, you must be redirected to the first page


## Screenshots

<table>
<body>
<tr>
<td>
<img src="https://user-images.githubusercontent.com/47038980/133837038-7d9f1aac-be0f-4ec2-bef1-7fcb6ee7e529.png" height="450" />
</td>
</tr>
</tbody>
</table>

## Types of changes




## Checklist



## Further comments
